src/style/_homepage.scss

```diff
@@ -0,0 +1,57 @@
+#HomePage {
+  height: 100vh;
+}
+
+.CityContainer {
+  display: flex;
+  height: 60%;
+  margin: 0 auto;
+  padding-left: 30px;
+  position: relative;
+}
+
+.CityForecast,
+.PinNewCity {
+  margin: 18px;
+  min-height: 50%;
+  width: 28%;
+}
+
+.CityForecast {
+  background-color: $color-forecast-background;
+  border: 1px solid darken($color-forecast-background, 50);
+  li {
+    margin-bottom: 10px;
+    padding: 10px;
+  }
+}
+
+.city-name {
```

Comment:
It seems like we're using these two properties a lot:

```css
margin-bottom: 10px;
padding: 10px;
```

You probably want to break this out into a mixin or at least a variable. What happens when you realize that 10px stays constant and doesn't look right for people who have increased the size of the font in their browsers (e.g. people with vision impairments and instructors at Turing)?